Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/datamgr/actions/CaptureFunctionDataTypesAction.java

```diff
@@ -110,9 +110,6 @@ private void informNotModifiable(ArchiveNode node) {
 		else if (node instanceof ProjectArchiveNode) {
 			message = "The data type archive \"" + node.getName() + "\" isn't checked out.";
 		}
-		else if (node instanceof ProjectArchiveNode) {
-			message = "The data type archive \"" + node.getName() + "\" isn't open for editing.";
-		}
```

Comment:
Sorry, I don't get what you want to change. The current behavior is useless, because we have exactly the same check before (connected with else), so this will never be reached, are you with me on that?

Do you want to change something else here?